.pre-commit-config.yaml

```diff
@@ -54,12 +31,19 @@ repos:
     description: Replace or check mixed line endings
   - id: trailing-whitespace
     description: Trim trailing whitespaces
-- repo: local
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.1.11
   hooks:
-  - id: style
-    name: Check style
-    description: Check style
-    pass_filenames: false
-    entry: tox -e project,flake8
```

Comment:
Did you remove `project` on purpose?

If we aren't using flake8 any longer, I'd also remove the respective tox config from `tox.ini`.

Reply:
As far as I can see in the code, project is just used to initialise the project to be able to run flake8. Not needed for any part of the tests if we remove the flake8 test. Yes, agreed, will remove from tox.ini